packages/mui-material/src/Backdrop/Backdrop.d.ts

```diff
@@ -50,17 +80,6 @@ export interface BackdropOwnProps extends Partial<Omit<FadeProps, 'children'>> {
    * If `true`, the component is shown.
    */
   open: boolean;
-  /**
-   * The extra props for the slot components.
-   * You can override the existing props or add new ones.
-   *
-   * This prop is an alias for the `componentsProps` prop, which will be deprecated in the future.
-   *
-   * @default {}
-   */
-  slotProps?: {
-    root?: React.HTMLAttributes<HTMLDivElement> & BackdropComponentsPropsOverrides;
-  };
```

Comment:
Thanks! We should also remove `slots` [below](https://github.com/mui/material-ui/pull/40677/files#diff-c4325a306ed3d9d31cbf99dfcc7f426e2e560e73af97357d36316f2950e10e00R90).